enhancements/windows-containers/container-runtime-containerd.md

```diff
@@ -64,10 +64,8 @@ will support Windows golden image with or without Docker.
 
 ### Containerd Migration plan
 
-Containerd will be the default runtime in OpenShift 4.11 for Windows. For development and testing during the
-OpenShift 4.10 / WMCO 5.0.0 cycle, we plan to introduce a WMCO CLI option which allows us to choose the runtime.
-This CLI option will be used in the release of the WMCO community operator for OpenShift 4.10 to allow customers
-to try out Windows nodes with containerd.
+Containerd will be the default runtime in OpenShift 4.11 for Windows. WMCO community operator for OpenShift 4.10 will
+have containerd as the default runtime to allow customers to try out Windows nodes with containerd.
```

Comment:
nit: Not just try out but actually use Windows nodes?

Reply:
rewrote and removed `try out`